CompuCell3D/core/CompuCell3D/ClassRegistry.cpp

```diff
@@ -11,7 +11,7 @@ ClassRegistry::ClassRegistry(Simulator *simulator) : simulator(simulator) {
 }
 
 
-Steppable *ClassRegistry::getStepper(string id) {
+Steppable *ClassRegistry::getStepper(string id) throw(CC3DException){

```

Comment:
With MSVC, these generate a lot of C4290 warnings.

Reply:
hmm interesting because without these clang does not compile the code at all... I did not have them I added them once clang started complaining. Apparently it is not a good idea to include exception specification  in function signature but this was one thing that got C++ exception working with Python when using Swig 4. HOw do you solve this issue in Mechanica @tjsego ? Can you take a stab at this?